service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -83,8 +84,15 @@ internal class RevurderingServiceImpl(
         }
 
         val dagensDato = LocalDate.now(clock)
-        if (!opprettRevurderingRequest.fraOgMed.isAfter(dagensDato.endOfMonth())) {
-            return KunneIkkeOppretteRevurdering.KanIkkeRevurdereInneværendeMånedEllerTidligere.left()
+        val startenAvForrigeKalenderMåned = dagensDato.minusMonths(1).startOfMonth()
+
+        val regulererGVerdiTilbakeITid =
+            revurderingsårsak.årsak == REGULER_GRUNNBELØP && !opprettRevurderingRequest.fraOgMed.isBefore(
+                startenAvForrigeKalenderMåned,
+            )
+
+        if (!regulererGVerdiTilbakeITid && !opprettRevurderingRequest.fraOgMed.isAfter(dagensDato.endOfMonth())) {
```

Comment:
Jeg tror det blir lettere å lese hvis vi lager en if for REGULER_GRUNNBELØP og en for de andre.